internal/pkg/clients/clients_test.go

```diff
@@ -108,9 +92,9 @@ func TestClients(t *testing.T) {
 		{},
```

Comment:
Add a test case for Gnosis, as it has different supported execution clients and consensus than mainnet

Reply:
solved at 6e38108

Reply:
I don't know how is that test not failing, gnosis doesn't support `prysm`. Check that again

Reply:
Check here please 6d2386a37497b6c648acfc3ec9e6132af8b67748